### PR TITLE
feat: add `Checkout` component to Playground

### PR DESCRIPTION
**Pull request by 0xAlec**
**What changed? Why?**
- add `Checkout` component to Playground
- add option for `chargeId` or `productId`

**Notes to reviewers**

**How has it been tested?**
locally + playground

### CODE REVIEW

File: playground/nextjs-app-router/components/DemoOptions.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        <>
+          {commonOptions}
+          <Chain />
+          <PaymasterUrl />
+          <TransactionOptions />
+        </>
+      );
+    default:
+      return commonOptions;
+  }
```
Comment (alessey) [inline playground/nextjs-app-router/components/DemoOptions.tsx:56]: 🥇

File: playground/nextjs-app-router/components/demo/Checkout.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+  const chargeIDDisabled = !checkoutOptions?.chargeId;
+  const productIDDisabled = !checkoutOptions?.productId;
+
+  return (
+    <div className="mx-auto grid w-1/2 gap-8">
+      {checkoutTypes === CheckoutTypes.ProductID && (
+        <>
+          {checkoutOptions?.productId ? (
+            <>
```
Comment (alessey) [inline playground/nextjs-app-router/components/demo/Checkout.tsx:38]: unnecessary fragments

**Review by alessey — APPROVED** [review]

File: playground/nextjs-app-router/components/demo/Checkout.tsx (hunk 2 above)
Comment (0xAlec) [reply playground/nextjs-app-router/components/demo/Checkout.tsx:38]: typescript was complaining, I think anything within () requires a parent div

Comment (alessey) [reply playground/nextjs-app-router/components/demo/Checkout.tsx:38]: yea, but Checkout should be your single parent

File: playground/nextjs-app-router/onchainkit/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@coinbase/onchainkit",
-  "version": "0.34.1",
+  "version": "0.35.0",
```
Comment (cpcramer) [inline playground/nextjs-app-router/onchainkit/package.json:3]: Need to update the `bun.lockb` file - we can just merge this pr though which has the version bump https://github.com/coinbase/onchainkit/pull/1493/files

**Review by cpcramer — APPROVED** [review]

**Pull request merged**